packages/formation-react/src/components/Sidebar/SidebarBlock.unit.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import SidebarBlock from './SidebarBlock';
+
```

Comment:
I'd like to add a unit test that passes a `level` prop "3" or "4" and ensure the rendered heading is correct.

Reply:
Added unit tests for prop of headingLevel 3 and 4 added